tests/Fixtures/Integration/priority/fully_qualified_strict_types,ordered_interfaces.test

```diff
@@ -0,0 +1,23 @@
+--TEST--
```

Comment:
nice one.

@Wirone , i would call it a bugfix than chore - without this change, the code requires 2 passes of fixing process, should only one - now it is only one. bugfix ;)

Reply:
You're right, I just overlooked it 🤷‍♂️.

Reply:
> without this change, the code requires 2 passes of fixing process

Not in this case. Usually, a PR of such kind is a bugfix, as it changes the priorities of the fixers, although, this one isn't technically a fix, as it does not change the priorities.

Reply:
good insight.
`test` then :D sooo many types